app/controllers/repos_controller.rb

```diff
@@ -26,10 +26,16 @@ def show
 
       @charts = @benchmark_type.benchmark_result_types.map do |benchmark_result_type|
         cache_key = "#{BenchmarkRun.charts_cache_key(@benchmark_type, benchmark_result_type)}:#{@benchmark_run_display_count}"
+        ruby_version_cache_key = "ruby_version_#{cache_key}"
 
-        if (columns = $redis.get(cache_key))
+        if (columns = $redis.get(cache_key)) && (ruby_versions = $redis.get(ruby_version_cache_key))
```

Comment:
This action (`repos#show`) is used for multiple repos, for instance Rails (https://rubybench.org/rails/rails/commits?result_type=actioncable/postgres&display_count=2000)

It looks like this controller part with `@ruby_versions` is only relevant for one Rubybench repo (ruby/ruby). We should make this cache universal for all repos. What do you think?